modules/ROOT/pages/performance-tuning.adoc

```diff
@@ -281,6 +281,20 @@ Two datasources are defined:
 
 Note that the sum of the maximum values configured for bonitaDS and bonitaSequenceManagerDS should be less than or equal to the maximum number of simultaneous connections allowed to your database.
 
+TIP: While the configuration of the 2 provided datasources works,
```

Comment:
You TIP doesn't embed the code example, is it what you add in mind?

Reply:
Up @danila-m